libr/anal/cond.c

```diff
@@ -69,6 +72,7 @@ static inline const char *condstring(RAnalCond *cond) {
 }
 
 R_API int r_anal_cond_eval(RAnal *anal, RAnalCond *cond) {
+	R_RETURN_VAL_IF_FAIL (anal && cond, false);
 	// XXX: sign issue here?
```

Comment:
We may probably honor the information in arg[0]/arg[1] which are RArchValue types. that contain the /absolute/ field to notify if the field is signed or unsigned. (i know its a pretty bad name, and its not even bool, but we may also fix that abi issue in after the upcoming release